api/SVGMarkerElement.json

```diff
@@ -2,6 +2,8 @@
   "api": {
     "SVGMarkerElement": {
       "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGMarkerElement",
+        "spec_url": "https://www.w3.org/TR/SVG2/painting.html#InterfaceSVGMarkerElement",
```

Comment:
```suggestion
        "spec_url": "https://svgwg.org/svg2-draft/painting.html#InterfaceSVGMarkerElement",
```

Reply:
No problem. Will do. 

The older was listed at browser-specs, though. Should (could?) we add a linter test that detect this?

Reply:
> The older was listed at browser-specs, though. Should (could?) we add a linter test that detect this?

I will think about this more. I didn't want to start with forbidding everything, but maybe we can get more strict :) browser-specs has a concept of nightly specs that are preferable and we can investigate how much non-nightly versions we have in BCD by tweaking the linter: https://github.com/mdn/browser-compat-data/blob/main/test/spec-urls.test.js#L16

Reply:
Thanks! Not urgent though. (I pushed the changes)